net/unbound/files/README.md

```diff
@@ -8,82 +8,170 @@ Unbound may be useful on consumer grade embedded hardware. It is *intended* to b
 
 This package builds on Unbounds capabilities with OpenWrt UCI. Not every Unbound option is in UCI, but rather, UCI simplifies the combination of related options. Unbounds native options are bundled and balanced within a smaller set of choices. Options include resources, DNSSEC, access control, and some TTL tweaking. The UCI also provides an escape option and work at the raw "unbound.conf" level.
 
-## Work with dnsmasq
-Some UCI options will help Unbound and dnsmasq work together in **parallel**. The default DHCP and DNS stub resolver in OpenWrt is dnsmasq, and it will continue to serve this purpose. The following partial examples will make Unbound the primary DNS server, and make dnsmasq only provide DNS to local DHCP.
+## Integrate with DHCP
+Some UCI options and scripts help Unbound to work with DHCP servers to load the local DNS. The examples provided here are serial dnsmasq-unbound, parallel dnsmasq-unbound, and unbound scripted with odhcpd.
+
+### Serial dnsmasq
+In this case, dnsmasq is not changed *much* with respect to the default OpenWRT/LEDE configuration. Here dnsmasq is forced to use the local Unbound instance as the lone upstream DNS server, instead of your ISP. This may be the easiest implementation, but performance degradation can occur in high volume networks. dnsmasq and Unbound effectively have the same information in memory, and all transfers are double handled.
+
+**/etc/config/unbound**:
+
+	config unbound
+		option add_local_fqdn '0'
+		option add_wan_fqdn '0'
+		option dhcp_link 'none'
+		# dnsmasq should not forward your domain to unbound, but if...
+		option domain 'yourdomain'
+		option domain_type 'refuse'
+		option listen_port '1053'
+		...
+
+**/etc/config/dhcp**:
+
+	config dnsmasq
+		option noresolv '1'
+		option resolvfile ''
+		option port '53'
+		list server '127.0.0.1#1053'
+		list server '::1#1053'
+		...
+
+### Parallel dnsmasq
+In this case, Unbound serves your local network directly for all purposes. It will look over to dnsmasq for DHCP-DNS resolution. Unbound is generally accessible on port 53, and dnsmasq is only accessed at 127.0.0.1:1053 by Unbound. Although you can dig/drill/nslookup remotely with the proper directives.
 
 **/etc/config/unbound**:
 
 	config unbound
-		option dnsmasq_link_dns '1'
```

Comment:
So from what I can gather dnsmasq_link_dns is going away altogether? Is there any setting replacing this or is it now default behaviour with unbound linking to dnsmasq for local DHCP-DNS?